docs/Tutorial.md

```diff
@@ -368,7 +368,7 @@ http://localhost:9684/?world=tutorial2
 
 Here is an image of the world we will be constructing:
 
-![Tutorial 2](./assets/tutorial2.png)
+![Tutorial 2](./docs/assets/Tutorial2.png)
```

Comment:
You can see this here:
[broken image](https://github.com/croquet/microverse-builder/blob/main/docs/Tutorial.md#tutorial-2)